lib/db/mysql/patches/patch-003-004.sql

```diff
@@ -0,0 +1,15 @@
+-- Adds support for Client Developers for OAuth clients
+
+CREATE TABLE IF NOT EXISTS clientDevelopers (
```

Comment:
I recommend removing `IF NOT EXISTS` here, so that if someone tries to replay the patch after it has already been applied, we get a noisy failure.  (But keep it in the schema.sql for creating from scratch)